content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -79,7 +79,35 @@ $ dvc exp run --all-pipelines
 > Note that the order in which pipelines are executed is not guaranteed; Only
 > the internal order of stage execution is.
 
-### Interactive reproduction
+### Running pipelines recursively
+
+DVC supports pipelines defined in more than one `dvc.yaml` file. These can
```

Comment:
Or a single dvc.yaml file can have multiple pipelines.

Reply:
Also each pipeline can be spread among several dvc.yaml files.

Let's find a way to describe all these possibilities in general without explaining each one here.

Reply:
Running recursively is about "more than one `dvc.yaml` file." Multiple pipelines is a different topic, and we may need to update `targets` discussion to include this possibility. I think introducing multiple pipelines here would be distracting to the base topic.

Reply:
If you're trying to avoid the topic of multiple pipelines you shouldn't use plural "pipelines" anywhere in this section. Not also in the text above but also bellow:

> To run the pipelines defined in `dir/dvc.yaml` as well as the ones in
`dir/deeper-dir/dvc.yaml` with a single command

and 

> It will run all the pipelines under `dir/`.

I agree we should focus on one thing or the other here so please make those singular pipeline.

Reply:
p.s. do we need an issue or check box about covering multiple pipelines? TBH probably not in the context of experiments.

Reply:
> we should focus on recursion only if possible

p.p.s. actually I'm not too sure about this. Probably the main use of `--recursive` is to run multiple pipelines... It makes sense to put a single pipeline in each dvc.yaml file. Splitting pipelines in different files seems much more advanced and we don't even have a single example of that throughout our docs.

Reply:
OK I thought about this. TBH I don't think we need this section here. The option just comes from `repro` and it will be documented in the pipelines guide as well as the `repro` and `exp run` cmd refs. This is not really about running experiments.

Reply:
I still this is not about running experiments at all. It's about how to structure data pipelines. I even plan to remove the concept of pipelines completely from this guide when possible, so there's no point in adding this. We merged the guide including that section for now for the sake of making progress.

Reply:
BTW we can reuse this content when we get to the pipelines guide so it won't be lost even if we close the PR, which is my recommended action.

Reply:
hmm, @jorgeorpinel why do you think that is not about experiments? If `dvc.yaml` files are independent then you have to specify explicitly or in some other way an entry point to run, right? `--recursive` can be considered as a way to specify it. 

I'm not sure about the details level for (e.g. do we need the whole subsection or not for this, etc), also not sure about the name. I would create a general section like "Specifying pipelines to run" and would name a few options there how to put a target, limit the scope, etc, etc when you run an experiment. To simplify we can even make a reference to other sections related to pipelines or whatnot.

WDYT, @jorgeorpinel @iesahin ?

Reply:
> If dvc.yaml files are independent then you have to specify explicitly or in some other way

OK, this is true. So do we want to cover this scenario in the guide as it's own section? But then maybe we need a separate "Organizing Experiments" page after all (instead of https://dvc.org/doc/user-guide/experiment-management#organization-patterns) and we mention it there as a tool. Or just mention `exp run --recursive` in that existing section for now (it can be a single sentence probably and link to the cmd ref for now).

I think the part about multiple or recursive pipelines is a topic for the Data Pipelines guide though.

> general section like "Specifying pipelines to run"

I think we should stop including the pipeline concept here since we plan to remove it from the experiments guide not to mix in concepts. Conceptually you don't need a pipeline (just technically a single stage that codifies some code) to run experiments.

Reply:
I think we should indeed try to avoid this, but I'm not sure why they could not be connected at all. Let's say we indeed have a repo with pipelines, some complicated set of pipelines (e.g. monorepo with a lot of models). When I run an experiment I understand that I have a lot of them and I need to specify something to run, right?

> Conceptually you don't need a pipeline (just technically a single stage that codifies some code) to run experiments.

You don't need != you don't use them. You don't need them for a "happy path", but if you use pipelines in the project you essentially run them when you run an experiment.

Reply:
Avoiding the concept doesn't mean they can not be connected. They can and if it's too hard to avoid that word then it's OK, we can include it and even link to Pipeline docs. I just think we should always be able to say "run the experiment" in this guide (focus on experimentation) and not "run pipelines". After all it's `exp run`, not `repro`.

Reply:
Fair enough. But I would not go into extremes here. I don't see we should be avoiding it completely. `exp` doesn't replace pipelines at all. They are two different things. And we do this section, the way I understand it is that it describes an advanced case - when you have a project with multiple pipelines and you do experiments with it.

Reply:
> BTW, I created https://github.com/iterative/dvc.org/issues/2883 (long awaited) so we can add check boxes there for things we want to cover related to pipelines.

Reply:
Moving this discussion to a new review as this one grew too much! See https://github.com/iterative/dvc.org/pull/2846#pullrequestreview-769287727 @iesahin @shcheklein thanks